dist/styles/base.scss

```diff
@@ -248,6 +248,11 @@
   }
 }
 
+// Include a "hidden option" class that can be placed on empty placeholder options to keep them out of the dropdown
+.hidden_option {
```

Comment:
I noticed all the other classes are prefixed with `fr_`, guessing thats a convention we might want to follow here.

Also, this is super minor, and I could be totally wrong, but I think we try to stick with semantic class naming. My guess is that `placeholder_option` might be more semantic than `hidden_option`. I usually think of semantic as "what is it" vs "how does it look"